cloudevents/spec.md

```diff
@@ -536,7 +536,7 @@ in the event directly.
 ## Privacy and Security
 
 Interoperability is the primary driver behind this specification, enabling such
-behavior requires some information to be made available _in the clear_ resulting
+behavior requires some information to be made available in the clear resulting
```

Comment:
@sunshengjing was the removal of the underscore on-purpose or an accident?

Reply:
it was an accident.

Reply:
sorry

Reply:
no worries - can you add it back?

Reply:
sure, I added it back. thanks